spec/controllers/cloud_network_controller_spec.rb

```diff
@@ -212,4 +212,30 @@
       end
     end
   end
+
+  describe '#button' do
+    before do
+      controller.instance_variable_set(:@edit, nil)
```

Comment:
Do you really need to set this to `nil`?

Reply:
No :D